{{ cookiecutter.__project_name_kebab }}/pyproject.toml

```diff
@@ -27,30 +27,27 @@ classifiers = [
     "Operating System :: OS Independent",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.7",
```

Comment:
This PR drops python 3.7 from the classifiers in the manifest but 3.7 is still in the tox matrix and the python versions in the CI build. What is the intended minimum compatible python version? 3.7 or 3.8? We should either consistently drop or retain 3.7 compatibility.

It is not a problem introduced in this PR, but this has also made me realise that this `pyproject.toml` does not include a `requires-python` declaration. Lets add an appropriate `requires-python` here while we're working on it.

Reply:
:+1:  Lets make sure we consistently ditch 3.7 then. Here, tox, GH workflows, all the places, and also add a `requires-python = ">=3.8"` to `pyproject.toml`.